app/scripts/routes.js

```diff
@@ -16,6 +16,10 @@ define([], function() {
                 templateUrl: 'views/students/index.html',
                 controller: 'StudentsIndexCtrl',
                 relativePath: '/students'
+            },
+            '/students/:docket/update': {
+                templateUrl: '/views/students/update.html',
```

Comment:
Note that views are now written without the starting slash. Fix this or it won't work on production environment.